references/video_classification/train.py

```diff
@@ -346,7 +351,13 @@ def get_args_parser(add_help=True):
     )
     parser.add_argument("--model", default="r2plus1d_18", type=str, help="model name")
     parser.add_argument("--device", default="cuda", type=str, help="device (Use cuda or cpu Default: cuda)")
-    parser.add_argument("--clip-len", default=16, type=int, metavar="N", help="number of frames per clip")
```

Comment:
One option could be to have `None` as default value and avoid passing 0s around. This will be nicer in terms of code IMO but as @datumbox pointed out in a conversation offline it has several downsides:
it doesn't play along with Batching, it causes issues with very short videos, etc